pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -1175,3 +1174,4 @@ https://github.com/ziglang/zig.vim/,,
 https://github.com/mickael-menu/zk-nvim/,HEAD,
 https://github.com/troydm/zoomwintab.vim/,,
 https://github.com/nanotee/zoxide.vim/,,
+https://git.sr.ht/~soywod/himalaya-vim,,
```

Comment:
I don't think you want this. If you update your Vim plugin for a major release and it's not merged yet to `nixpkgs`'s default branch (happens a lot, even if approved and ready), then downstream users will get into a broken state as the Vim plugins get updated on a regular schedule. You could either tack on a branch like `v1.x` that it's safe to pull updates from and not break, or it is manually updated in the `overrides.nix` in the Vim plugins folder.

Reply:
For context, I originally added it in here _because_ getting feedback and a merge took too long. It'd be a good idea to leave a note/comment in the `default.nix` though to double check the Vim plugin version or something.

Reply:
I see. I will propose sth and let you know!